src/components/registration-lite.js

```diff
@@ -420,6 +431,12 @@ RegistrationLite.defaultProps = {
     authErrorCallback: (error) => { console.log(error) },
     hasVirtualAccessLevel: false,
     supportEmail : 'support@fntech.com',
+    initialOrderComplete1stParagraph: '',
```

Comment:
this is not correct
u cant define default values here
please check the strategy used here
https://github.com/fntechgit/summit-registration-lite/blob/2d8a39f6f09729c69131c526732ccc6bff598fcc/src/components/registration-lite.js#L372